gammapy/spectrum/models.py

```diff
@@ -26,6 +27,110 @@ def __call__(self, energy):
         """Call evaluate method of derived classes"""
         return self.evaluate(energy, **self.parameters)
 
+    def __str__(self):
+        """String representation"""
+        ss = self.__class__.__name__
+        for parname, parval in self.parameters.items():
+            ss += '\n{parname} : {parval:.3g}'.format(**locals())
+        return ss
+
+    def to_sherpa(self, name='default'):
+        """Return `~sherpa.models.ArithmeticModel`
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the sherpa model instance
+        """
+        import sherpa.models as m
+        if isinstance(self, PowerLaw):
```

Comment:
I think having `to_sherpa` in the base class isn't a good idea, because it's not nicely extensible and not nice to maintain.

Extensible: What if Fermipy or Naima or end users want to define their own spectral models and fit them with Gammapy? Having to edit this file in Gammapy or monkey-patch it is bad.
So they'd have to define their own `to_sherpa` method. It's nicer if others can define new models in the same way that the built-in models are defined.

Maintain: Having to touch the base class every time a new model is added isn't as nice compared to the alternative: just add a new class, don't touch any other code in Gammapy.